### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by keith** [MobileNativeFoundation/discussions#5] [General]
Welcome to the Mobile Native Foundation’s discussions repo! This forum can be used for public, searchable discussions for anything related to mobile infrastructure. We hope this can become a go-to place to discuss problems companies face while developing mobile apps at scale.

Have something you want to discuss? Start [here](https://github.com/MobileNativeFoundation/discussions/discussions/new), or checkout the [existing discussions](https://github.com/MobileNativeFoundation/discussions/discussions)

As a friendly reminder the Mobile Native Foundation is governed by the [code of conduct](https://lfprojects.org/policies/code-of-conduct/).

**Comment by freak4pc** [comment]
Thanks for setting this up!

**Reply by aln787** [reply]
+1

**Comment by joshwoods** [comment]
I think this is a great idea. I definitely look forward to the conversations that will come out of here!

**Reply by crismaver1993** [reply]
Thank you so much for this initiative.

**Comment by kazimdsaidul** [comment]
Thanks for good initiative!!!
